spec/features/advanced_searching_spec.rb

```diff
@@ -42,6 +42,14 @@
     expect(page).not_to have_content("Огонек : роман")
   end
 
+  it 'allows searching by publication date', js: true do
```

Comment:
This test was to catch a regression I made in the advanced search form while working on the `render_range_input` deprecation.